app/tsconfig.json

```diff
@@ -2,21 +2,10 @@
   "extends": "../base.tsconfig.json",
   "include": [
     "src",
+    "./webpack.config.js"
```

Comment:
Why do we need it?

Reply:
It's not mandatory, but I think it's better to have it monitor by ts so we can have monitored

Reply:
Removed it for now